README.md

```diff
@@ -168,6 +168,25 @@ The options for `phylum-ci`, automatically updated to be current for the latest
 [jenkins_docs]: https://docs.phylum.io/phylum-ci/jenkins
 [precommit_docs]: https://docs.phylum.io/phylum-ci/git_precommit
 
+### Exit Codes
+
+The `phylum-init` script entry point will return a zero (0) exit code when it completes successfully and a one (1)
+otherwise.
+
+The `phylum-ci` script entry point will also return a zero (0) exit code when it completes successfully or one of the
```

Comment:
```suggestion
The `phylum-ci` script entry point will return a zero (0) exit code when it completes successfully or one of the
```

This "also" seems like it just makes the sentence harder to read.

Reply:
That change was made.